kdumpctl

```diff
@@ -1668,6 +1672,137 @@ _should_reset_crashkernel() {
 	[[ $(kdump_get_conf_val auto_reset_crashkernel) != no ]] && systemctl is-enabled kdump &> /dev/null
 }
 
+clear_kdump_test_status()
+{
+	echo > $KDUMP_STATUS
+}
+
+# Get the latest dump's info from the dump target in the format:
+# /var/crash 127.0.0.1-2024-05-01-15:54:29/vmcore 1714550071
+#
+# return: 0 on success, 1 on failure
+get_latest_dump()
+{
+	local _latest_record _target
+	local _path=$(get_save_path)
+	if is_raw_dump_target; then
+		dinfo "Unable to check for raw dump. Manual verification of vmcore collection required."
+		return 1
+	elif is_ssh_dump_target; then
+		_latest_record=$(ssh -i "${OPT[sshkey]}" -o BatchMode=yes "${OPT[_target]}" \
```

Comment:
If there are multiple machine dumping to same ssh/nfs server, then just finding the latest timestamp of the vmcore will be not enough here..